src/platforms/javascript/common/configuration/micro-frontend-support.mdx

```diff
@@ -129,18 +129,7 @@ You can then set tags/contexts on events in individual micro-frontends to decide
 containing the DSN and optionally the release.
 
 ```typescript
-type EnvelopeItemType =
-  | "client_report"
-  | "user_report"
-  | "session"
-  | "sessions"
-  | "transaction"
-  | "attachment"
-  | "event"
-  | "profile"
-  | "replay_event"
-  | "replay_recording"
-  | "check_in";
+import { Envelope, EnvelopeItemType } from "@sentry/types";
```

Comment:
I feel it's better to move the type declaration from [here](https://docs.sentry.io/platforms/javascript/guides/react/configuration/micro-frontend-support/#makemultiplexedtransport-api) to [one section above](https://docs.sentry.io/platforms/javascript/guides/react/configuration/micro-frontend-support/#routing-events-to-different-projects).  Because, when I was implementing `dsnFromFeature` from [routing events to different projects section](https://docs.sentry.io/platforms/javascript/guides/react/configuration/micro-frontend-support/#routing-events-to-different-projects), I was trying to figure out what `type` of argument it is expecting.

What are your thoughts?

Reply:
I think it's a good idea to make that change, let's move it!